pystac/html/Asset.jinja2

```diff
@@ -0,0 +1,31 @@
+{% import 'Macros.jinja2' as macros %}
```

Comment:
We already have a `package_data` argument in `setup.py` to include the `py.typed` file, so that would just need to be updated to include these files as well. Something as general as `package_data={"": ["py.typed", "*.jinja2"]}` would probably be fine.